v2/pkg/protocols/http/httpclientpool/clientpool.go

```diff
@@ -210,14 +209,13 @@ func wrappedGet(options *types.Options, configuration *Configuration) (*retryabl
 			transport.Proxy = http.ProxyURL(proxyURL)
 		}
 	} else if types.ProxySocksURL != "" {
-		var proxyAuth *proxy.Auth
 		socksURL, proxyErr := url.Parse(types.ProxySocksURL)
-		if proxyErr == nil {
-			proxyAuth = &proxy.Auth{}
-			proxyAuth.User = socksURL.User.Username()
-			proxyAuth.Password, _ = socksURL.User.Password()
+		dialer, err := proxy.FromURL(socksURL, proxy.Direct)
+		if err != nil {
+			gologger.Error().Msgf("Unable to setup socks proxy: %s\n", err)
```

Comment:
instead of logging here we can just return the error and it will be logged anyway